### PR TITLE
Release/0.17.1

### PR DESCRIPTION
**Pull request by DmitriyMolch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # CHANGELOG
 
 ## vNext
-- ...
+...
+
+## 0.17.1
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md:6]: You should increase minor and not patch.

Comment (DmitriyMolch) [reply CHANGELOG.md:6]: This is a minor version on the 2nd place

Comment (AndreiNavoichyk) [reply CHANGELOG.md:6]: Ok, you've increased minor and patch. Please, correct the version to `0.17.0`

Comment (DmitriyMolch) [reply CHANGELOG.md]: fixed

**Pull request closed without merging**